code/modules/mob/living/carbon/resist.dm

```diff
@@ -107,6 +107,9 @@
 	if(HULK in mutations)
 		return 1
 
+	if(stamina > 100)

```

Comment:
Shouldnt that be < 100 ?
(You want it to fail if they dont have enough stamina left ?)